src/api.ts

```diff
@@ -17,8 +17,9 @@ export const BASE_AUTH = `${BASE_URL}/Users`;
 export const AUTH_URLS = {
   login: ``,
   register: `/Register`,
-  verify: ``,
+  verify: `/verify`,
   forgetPassword:`${BASE_AUTH}/Reset/Request`,
```

Comment:
We need to be consistent here, if we will not use the base auth, we need to remove it.

Reply:
done